.github/CONTRIBUTING.md

```diff
@@ -61,6 +61,10 @@ To run the spellchecker, follow the steps as follows:
 
 ## Contributing to Code
 
+### Code Editors
+
+[You can use multiplatform Visual Studio Code (VS Code)][use-vscode-editor]
```

Comment:
I think it should say: You should use the multi-platform ...

And make just the `Visual Studio Code (VSCode)` the hyperlink instead of the whole line